pluto-message-ingestion/package.json

```diff
@@ -17,8 +17,8 @@
     "build": "npm run test && npm run build-artefact"
   },
   "dependencies": {
-    "crypto": "0.0.3",
     "@guardian/elk-kinesis-logger": "0.1.0",
+    "crypto": "0.0.3",
```

Comment:
Should this value have changed?

Reply:
I think it just reorders it so it's alphabetical?